font.go

```diff
@@ -70,7 +70,30 @@ func (f *Font) Raw() (string, []byte) {
 	return f.mimetype, f.raw
 }
 
-func (f *Font) pdfInfo() (Rect, float64, float64, float64, float64, []int) {
+// UnitsPerEm returns the number of units per em for f.
+func (f *Font) UnitsPerEm() float64 {
+	return float64(f.sfnt.UnitsPerEm())
+}
+
+// Kerning returns the horizontal adjustment for the rune pair. A positive kern means to move the glyphs further apart.
+func (f *Font) Kerning(left, right rune, ppem float64) (float64, error) {
+	var sfntBuffer sfnt.Buffer
+
+	iLeft, err := f.sfnt.GlyphIndex(&sfntBuffer, left)
+	if err != nil {
+		return 0, err
+	}
+	iRight, err := f.sfnt.GlyphIndex(&sfntBuffer, right)
+	if err != nil {
+		return 0, err
+	}
+
+	kern, err := f.sfnt.Kern(&sfntBuffer, iLeft, iRight, toI26_6(ppem), font.HintingNone)
```

Comment:
What I mean is that it is not clear what `kern` is when `err != nil`. Apparently `Kern` returns zero if it has an error, maybe add a comment to clarify?

Reply:
I added a comment at the top of the method